lib/SILGen/SILGenLValue.cpp

```diff
@@ -4411,6 +4411,13 @@ LValue SILGenLValue::visitBindOptionalExpr(BindOptionalExpr *e,
     
     if (optBase.getType().isMoveOnly()) {
       if (optBase.getType().isAddress()) {
+        // Strip the move-only wrapper if any.
+        if (optBase.getType().isMoveOnlyWrapped()) {
+          optBase = ManagedValue::forBorrowedAddressRValue(
+            SGF.B.createMoveOnlyWrapperToCopyableAddr(e,
+                                                      optBase.getValue()));
+        }
```

Comment:
huh, i thought move-only types aren't suppose to appear in that wrapper.

Reply:
`SILType::isMoveOnly` includes move-only-wrapped copyable types along with always-move-only types.